platform_versioned_docs/version-24.1/compute-envs/azure-batch.mdx

```diff
@@ -11,113 +11,162 @@ For details, visit [Azure Free Account][az-create-account].
 Ensure you have sufficient permissions to create resource groups, an Azure Storage account, and a Batch account.
 :::
 
-## Concepts
+## Azure concepts
```

Comment:
My 10 cents here is that the boxes are quite intrusive and distracting. When I first open this page all I see are these boxes. I would have this text shown under headings rather than as boxes.